internal/venice-client-common/src/main/java/com/linkedin/venice/schema/writecompute/WriteComputeSchemaConverter.java

```diff
@@ -446,26 +443,4 @@ public Schema getNoOpOperation(String namespace) {
     noOpSchema.setFields(Collections.emptyList());
     return noOpSchema;
   }
-
-  public static WriteComputeOperation getFieldOperationType(Object writeComputeFieldValue) {
-    Utils.notNull(writeComputeFieldValue);
```

Comment:
So I guess the fix is that the copy of this function in `WriteComputeOperation` skips this `notNull` check. This is fine. `null instanceof blah` returns false, so the rest of the code is good even without this null check.

Reply:
Yeah this is true.